### PR TITLE
add .net sdk tests for rotationToLatestKeyVersionEnabled

### PR DESCRIPTION
**Pull request by parkhyukjun89**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs [hunk 1]
```diff
@@ -337,6 +337,53 @@ protected void DiskEncryptionSet_CRUD_Execute(string methodName, string encrypti
...
+                    // Put DiskEncryptionSet
+                    DiskEncryptionSet desOut = m_CrpClient.DiskEncryptionSets.CreateOrUpdate(rgName, desName, des);
+                    Validate(des, desOut, desName, encryptionType, expectedRotationToLatestKeyVersionEnabled);
+
+                    // Get DiskEncryptionSet
+                    desOut = m_CrpClient.DiskEncryptionSets.Get(rgName, desName);
+                    Validate(des, desOut, desName, encryptionType, expectedRotationToLatestKeyVersionEnabled);
+                    
+                    // Delete DiskEncryptionSet
+                    m_CrpClient.DiskEncryptionSets.Delete(rgName, desName);
```
Comment (parkhyukjun89) [inline sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs:366]: There is no update test as updating DES with rotationToKeyVersionEnabled set to true triggers key rotation, which requires real Key Vault key and access policy in place

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPDiskEncryptionSetTests.cs [hunk 2]
```diff
@@ -43,5 +43,11 @@ public void DiskEncryptionSet_AddDESToExistingDisk()
         {
             DiskEncryptionSet_UpdateDisk_Execute("DiskEncryptionSet_AddDESToExistingDisk", location: "centraluseuap");
         }
+
+        [Fact]
+        public void DiskEncryptionSet_CRD_RotationToLatestKeyVersionEnabled()
```
Comment (laurawu19) [inline sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPDiskEncryptionSetTests.cs:48]: DiskEncryptionSet_CRUD_RotationToLatestKeyVersionEnabled

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs (hunk 1 above)
Comment (laurawu19) [reply sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs:366]: Can we update tags for diskEncryptionSet when rotationToKeyVersionEnabled set to true?

Comment (parkhyukjun89) [reply sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs:366]: Any update with flag set to true will trigger the rotation.

**Review by vikramd-ms — APPROVED** [review]

**Pull request merged**